consensus/ibft/sign.go

```diff
@@ -88,12 +88,12 @@ func writeCommittedSeals(h *types.Header, seals [][]byte) (*types.Header, error)
 	h = h.Copy()
 
 	if len(seals) == 0 {
-		return nil, fmt.Errorf("bad")
+		return nil, fmt.Errorf("empty committed seals")
 	}
 
 	for _, seal := range seals {
 		if len(seal) != IstanbulExtraSeal {
-			return nil, fmt.Errorf("bad")
+			return nil, fmt.Errorf("invalid committed seal")
```

Comment:
Just a bit more descriptive 🙂 
```suggestion
			return nil, fmt.Errorf("invalid committed seal length")
```